tests/accumulator_test.py

```diff
@@ -262,18 +263,60 @@ def test_without_noise(self):
 class SumAccumulatorTest(unittest.TestCase):
 
     def test_without_noise(self):
+        no_noise = MeanVarParams(eps=1,
+                                 delta=1,
+                                 low=0,
+                                 high=0,
+                                 max_partitions_contributed=1,
+                                 max_contributions_per_partition=1,
+                                 noise_kind=pipeline_dp.NoiseKind.GAUSSIAN)
         sum_accumulator = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(6)))
+            accumulator.SumParams(no_noise), list(range(6)))
         self.assertEqual(sum_accumulator.compute_metrics(), 15)
 
         sum_accumulator.add_value(5)
         self.assertEqual(sum_accumulator.compute_metrics(), 20)
 
         sum_accumulator_2 = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(3)))
+            accumulator.SumParams(no_noise), list(range(3)))
         sum_accumulator.add_accumulator(sum_accumulator_2)
         self.assertEqual(sum_accumulator.compute_metrics(), 23)
 
+    def test_with_noise(self):
+        sum_accumulator = accumulator.SumAccumulator(
+            accumulator.SumParams(
+                MeanVarParams(eps=100,
+                              delta=1,
+                              low=1,
+                              high=1,
+                              max_partitions_contributed=1,
+                              max_contributions_per_partition=3,
+                              noise_kind=pipeline_dp.NoiseKind.GAUSSIAN)),
+            list(range(6)))
+        self.assertAlmostEqual(first=sum_accumulator.compute_metrics(),
+                               second=15,
+                               places=1)
+
+        sum_accumulator.add_value(5)
+        self.assertAlmostEqual(first=sum_accumulator.compute_metrics(),
+                               second=20,
+                               places=1)
```

Comment:
Please use delta=1.0 or delta = 0.5, instead of places = 1. Here and in other places

Reply:
After making the other changes I had to make the delta 4 to have the test pass consistently. Is that okay or should we do more parameter tweaking?